docs/source/tutorial/tutorial.rst

```diff
@@ -312,8 +312,8 @@ using the LidGroup object.
 	... 		print(lid)
 	... 	print(lid_on_sub[0])
 	... 	for step in sim:
-	... 		print(lid_on_sub.WaterBalance.inflow)
-	... 		print(lid_on_sub.WaterBalance.evaporation)
+	... 		print(lid_on_sub.water_balance.inflow)
```

Comment:
@dokzai does this need to be `print(lid.water_balance.inflow)`? Are you pulling the water balance from the group? or the actual LID unit?